more_itertools/recipes.py

```diff
@@ -304,26 +304,42 @@ def powerset(iterable):
 
 
 def unique_everseen(iterable, key=None):
-    """Yield unique elements, preserving order.
```

Comment:
Have we shipped this function yet? If not, we should rename it unique_ever_seen.